training/src/training_dataset_watchdog.py

```diff
@@ -0,0 +1,54 @@
+import os
+import time
+import constants
+from training import train_model
+from embeddings import extract_embeddings
+import logging
+import schedule
+import log_config
+
+# Initializing custom logger
+log = logging.getLogger('root')
+log.setLevel(logging.INFO)
+log.addHandler(log_config.LogHandler())
+
+log = logging.getLogger('root')
+log.info("Watchdog started.")
+
+def job():
+    already_trained_students_count = -1
+    students_image_count_obj = {}
+    try:
+        for username in os.listdir(constants.DATASET_PATH):
+            concatenated_dir = constants.DATASET_PATH + '/' + username
+            if os.path.isdir(concatenated_dir):
+                students_image_count_obj[username] = len([file for file in os.listdir(concatenated_dir) if (
+                            os.path.isfile(concatenated_dir + "/" + file) and os.path.splitext(file)[
+                        1].lower() == ".png")])
```

Comment:
Does streamProcessor generate images of png type?
Can we move this to supportedFileTypes property in the config?

Reply:
Yes, Images generated by streamProcessor are of .png extension.
supportedFileTypes -> do you want it to be a list of images extension or just png?

Reply:
Cool. For now, we can add a config entry **supportedFileType** with a single value **png**